server/views/pages/induction/prePrisonEducation/qualificationLevel.njk

```diff
@@ -19,6 +19,17 @@ Data supplied to this template:
 {% endblock %}
 
 {% block content %}
+  <div class="govuk-grid-row">
```

Comment:
Because of the required layout, where the sidebar is horizontally in line with the top of the first radio button, I've had to move the form legend (page title) and hint text out of the `govukRadios` `fieldset` and essentially hardcode the markup for them here in their own row.